content/docs/introduction/getting_started.md

```diff
@@ -86,6 +86,10 @@ Blindly setting `GOMAXPROCS` to a high value can be
 counterproductive. See the relevant [Go
 FAQs](http://golang.org/doc/faq#Why_no_multi_CPU).
 
+Notice that Prometheus by default uses around 3GB in memory. If you have a
```

Comment:
Ah forgot this in the first round, but "Notice that" would be better as "Note that".